controllers/topics.controllers.js

```diff
@@ -0,0 +1,15 @@
+const { fetchTopics } = require("../models/topics.models");
+
+const getTopics = (req, res, next) => {
+  fetchTopics()
+    .then((topics) => {
+      res.status(200).send({ topics });
+    })
+    .catch((err) => {
+      if (err) {
+        res.status(500).send({ msg: "Internal Server Error" });
+      }
```

Comment:
We shouldn't be dealing with this within the catch block, we want to invoke next() with the error and building error handling middleware